front/init/init.sh

```diff
@@ -1,6 +1,3 @@
 #!/bin/sh
+env $(cat .env.local) npx tsx init/db.ts
```

Comment:
cc @spolu WDYT of this ?

Reply:
Looks great. As long as it works I’m cool with it 👍